### ISSUE TITLE
MLC dump is missing Wii VC *.nfs files

### ISSUE THREAD

**Issue by the16bitgamer** [emiyl/dumpling#19]
I am trying to extract the Wii VC *.nfs files with dumpling, however the *.nfs files are not being included with the MLC dump for any of the Wii VC titles.

**Comment by Crementif** [comment]
Yeah, unfortunately there's some extra protection on Wii VC games. I hope to be able to eventually figure out why it's like that, but in the meantime you could dump your online files and use Cemu's Tools->Download Manager to download the Wii games you've bought via the eshop if you wanted to. There's a guide that shows how here: https://cemu.cfw.guide/dumping-games.html?tab=digital-titles. Can right-click the game in Cemu's game list to open the game directory to be able to access the  .nfs files.

**Comment by Crementif** [comment]
With the latest release it's now possible to dump vWii! Let me know if it works properly if you haven't gotten it to work through other means.

**Comment by the16bitgamer** [comment]
Fantastic news, just updated my Dumpling and other than a small issue which required a reboot to fix (couldn't read a meta file?) I am now backing up my Wii eShop games (2h 30m) I will create a new issue if I encounter anything that is repeatable.

**Comment by Crementif** [comment]
Yeah, other people also noticed that disc dumping isn't possible with the new release. I looked into it for a while but can't find the bug, and unfortunately don't have time before I'm off to my holidays, so I've decided to redact the release for now. Just as an update if you're wondering where the release went.

**Comment by Crementif** [comment]
I'm back and have just released version 2.4.0b which should have addressed the issue with the previous release! Let me know if you can replicate/see any issues with it if you happen to try using it, but happy to have created something useful for you.

**Comment by the16bitgamer** [comment]
Just finished testing the update now. Limited testing since I got all my eShop Wii games with the last update. However I was able to dump Super Mario Galaxy 2 without issue. As well as dumping the Physical version of Wonderful 101 immediately afterward.

It's a shame that there is currently no way to re-install dumpling backups onto the Wii U, but I know that's out of scope for the application.

Thank you for adding in the ability to dump eShop Wii games.
